WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs

```diff
@@ -27,13 +27,15 @@ public WasabiJsonRpcService(Global global)
 		public object[] GetUnspentCoinList()
 		{
 			AssertWalletIsLoaded();
+			var tipHeight = ActiveWallet.BitcoinStore.SmartHeaderChain.TipHeight;
 			return ActiveWallet.Coins.Where(x => x.Unspent).Select(x => new
 			{
 				txid = x.TransactionId.ToString(),
 				index = x.Index,
 				amount = x.Amount.Satoshi,
 				anonymitySet = x.AnonymitySet,
 				confirmed = x.Confirmed,
+				confirmations = tipHeight - x.Height + 1,
```

Comment:
Umm I think the number of confirmations should be `ServerTipHeight - x.Height` because `TipHeight` is the height of your wallet during the synchronization process but the real number of blocks mined on top of the one containing the transaction depends on the height of the network (we called Server).

Please make sure to test it with transactions waiting on the mempool too because i think x.Height is a `Height` instance and tip is `uint` so, not sure how these play together.

Reply:
Updated to PR to address these suggestions. Thanks @lontivero !